### ISSUE TITLE
problem at getting started

### ISSUE THREAD

**Issue by posseidon23** [coin-or/CyLP#36]
Hi, here is what get when running : >>>from cylp.cy import CyClpSimplex   

```
Traceback (most recent call last):
File "<stdin>", line 1, in <module>
 File "cylp/cy/__init__.py", line 1, in <module>
  from CyCoinIndexedVector import CyCoinIndexedVector
 ImportError: No module named CyCoinIndexedVector
```

cannot figure out why getting this message!

**Comment by sschnug** [comment]
How did you install it?
It looks, that you did something wrong with defining the path-variables.

**Comment by posseidon23** [comment]
I followed exactly the instruction given here on GitHub.

**Comment by sschnug** [comment]
Are you by any chance using python 3? There is pull-request which is the only way for now if you are using py3.

**Comment by posseidon23** [comment]
it is python 2.7.11

**Comment by sschnug** [comment]
I just tested it with a fresh python2 install and it worked for me! No problems at all.

- Ubuntu-based system
- Python 2.7.11 |Continuum Analytics, Inc.| (default, Dec  6 2015, 18:08:32) [GCC 4.4.7 20120313 (Red Hat 4.4.7-1)] on linux2
- Cbc-2.9.7 (officially only Cbc-2.8.5 is supported)

You need to be way more precise on your tools and steps performed. Otherwise it's hard to help!

**Comment by snitramodranoel** [comment]
Hi all, I'm having exactly the same problem on OSX Yosemite 10.10.5 with Python 2.7.12 in a `pyenv` silo. I managed to compile Cbc-2.9.8 (I just can't get 2.8.5 to compile) without any extra `./configure` parameters. CyLP compiles without any complaints, except for the following `ld` warning:

> ld: warning: directory not found for option '-L/Users/username/cbc-2.9.8/lib/intel

Please, let me know if you need any more information.

**Comment by tkralphs** [comment]
This directory is only needed for Windows. Although this is only a warning and shouldn't affect anything, I changed `setup.py` so that it only gets added when the OS is detected as Windows. This change is [here](https://github.com/coin-or/CyLP/tree/elim_warning). Can you check out this branch and see if everything seems OK? If so, I will merge into master.

**Comment by snitramodranoel** [comment]
@tkralphs No more warnings on my system. Thanks.

**Comment by snitramodranoel** [comment]
On the original note, though, I still get the same import error:

`In [1]: from cylp.cy import CyClpSimplex`
`---------------------------------------------------------------------------`
`ImportError                               Traceback (most recent call last)`
`<ipython-input-1-699be7eca712> in <module>()`
`----> 1 from cylp.cy import CyClpSimplex`

`/Users/leonardo/Build/CyLP/cylp/cy/__init__.py in <module>()`
`----> 1 from CyCoinIndexedVector import CyCoinIndexedVector`
`      2 from CyClpSimplex import CyClpSimplex`
`      3 from CyCbcNode import CyCbcNode`
`      4 from CyClpPrimalColumnPivotBase import CyClpPrimalColumnPivotBase`
`      5 from CyCoinMpsIO import CyCoinMpsIO`

`ImportError: No module named CyCoinIndexedVector`

**Comment by tkralphs** [comment]
I just verified that everything works fine for me on OS X, but there are so many ways to build on OS X, so you'll have to give a little more detail and do some digging yourself. Are you sure that the installation was successful in the end? What are the last few lines of the output of the setup command? The proximal cause of the import error should be pretty easy to ascertain and then we can go from there.

**Comment by samiit** [comment]
I could reproduce the error of @posseidon23 and @snitramodranoel in the following way:

1. Install CyLP as instructed in its folder - /home/CyLP
2. Started python shell in the same folder
3. `from cylp.cy import CyClpSimplex` throws the error as described

I think it is because you start the Python REPL in the folder where the `build` directory exists. If you do this in any other folder, you can import without these errors.

Actually the error messages from @snitramodranoel  also show which `__init__.py` is being accessed! If the installation worked well, CyLP should be placed in the dist_packages or site_packages folder and then things should work just fine.

Sam

**Comment by roversch** [comment]
I get a related error, on Mac OSX 10.12.4, with Cbc2.8.5 built from source and `cylp` installed with `python setup.py install`, both without any errors whatsoever.

```
ImportError                               Traceback (most recent call last)
<ipython-input-1-a93bc4cafb27> in <module>()
----> 1 import cylp.cy

/Users/robin/miniconda3/envs/cvxpy1.0/lib/python2.7/site-packages/cylp/cy/__init__.py in <module>()
----> 1 from CyCoinIndexedVector import CyCoinIndexedVector
      2 from CyClpSimplex import CyClpSimplex
      3 from CyCbcNode import CyCbcNode
      4 from CyClpPrimalColumnPivotBase import CyClpPrimalColumnPivotBase
      5 from CyCoinMpsIO import CyCoinMpsIO

ImportError: dlopen(/Users/robin/miniconda3/envs/cvxpy1.0/lib/python2.7/site-packages/cylp/cy/CyCoinIndexedVector.so, 2): Symbol not found: __ZNK9CoinParam7matchesESs
  Referenced from: /Users/robin/miniconda3/envs/cvxpy1.0/lib/python2.7/site-packages/cylp/cy/../cbclibs/lib.macosx-10.7-x86_64-2.7/libCoinUtils.3.dylib
  Expected in: flat namespace
 in /Users/robin/miniconda3/envs/cvxpy1.0/lib/python2.7/site-packages/cylp/cy/../cbclibs/lib.macosx-10.7-x86_64-2.7/libCoinUtils.3.dylib
```

**Comment by tkralphs** [comment]
Fixed by #50.
